xcpmd/src/db-helper.c

```diff
@@ -865,6 +865,18 @@ struct db_var * add_var(char * name, enum arg_type type, union arg_u value, char
 
     if (var != NULL) {
         write_db_var(var->name, var->value.type, var->value.arg);
+
+        if (db_event_table != NULL) {
+            e = db_event_table[EVENT_VAR_WRITTEN];
```

Comment:
EVENT_VAR_WRITTEN and EVENT_VAR_WRITTEN_EDGE are only defined in the next commit:
https://github.com/OpenXT/xctools/pull/74/commits/80eff6c86926b07ced730e72a6ddce9a0eef5b29

Reply:
I folded this into the "Add db-events module" commit, as they likely should have been there in the first place.  The refresh changes below will remain in this commit.